### ISSUE TITLE
All Pages - Missing Skip Navigation Link

### ISSUE THREAD

**Issue by jfc3-dol** [presidential-innovation-fellows/worker-dol-gov#15]
While testing using the DHS Trusted Tester process, I found all the pages were missing a skip link to allow screen reader and keyboard only users to skip over the redundant content at the top of the page such as the banner area and WORKER.gov logo.

Section 508 - 22(o)
WCAG 2.0 - 13 Web: Repetitive Content and Links

**Comment by adamsfallen** [comment]
So right now there are only 3 or so "navigation" links (logo + disclaimer) that one would have to skip over, so I'm a bit hesitant to add new links for everybody that would be only a *tiny* bit helpful to screen readers. Thoughts?

And if you do think it's worth considering, would you recommend ways of implementing that are unobtrusive to non-screen-readers?

**Comment by jfc3-dol** [comment]
Sorry, I should have said that you can visually hide the link off screen until a person tabs to it.

If you look at my personal website ( http://jfciii.com ) you can see an example of a hidden skip navigation until you tab to it.

![worker dot gov - skip navigation example](https://cloud.githubusercontent.com/assets/10423487/19774226/5e5f1564-9c3a-11e6-8d48-440ea6d900d2.png)

You need to add a link directly after the BODY tag that anchors down the page where the main content starts. An example would be:

<body>
<div class="scrn-rdr">
	<a href="#maincontent">Skip to Main Content</a>
</div>

CODE FOR HEADER, BANNER, ETC. GOES HERE

<div id="maincontent">

MAIN CONTENT...

The CSS I use is the following:

.scrn-rdr a, .scrn-rdr a:hover, .scrn-rdr a:visited { 
display: block;
position: absolute;
left: -10000px;
top: auto;
width: 100%;
height: 3em;
font-weight: bold;
text-align: center;
border-bottom:
1px solid #000;
} 

.scrn-rdr a:active, .scrn-rdr a:focus { 
background: #ccc;
position: static;
width: auto;
height: auto;
padding: 1em;
margin: auto;
}
